test/e2e/metrics_k8s_cluster_receiver_test.go

```diff
@@ -0,0 +1,154 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"io"
+	"net/http"
+	"slices"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/telemetry-manager/internal/otelcollector/config/metric"
+	"github.com/kyma-project/telemetry-manager/internal/testutils"
+	"github.com/kyma-project/telemetry-manager/test/testkit/assert"
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	. "github.com/kyma-project/telemetry-manager/test/testkit/matchers/metric"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/otel/k8scluster"
+	"github.com/kyma-project/telemetry-manager/test/testkit/otel/kubeletstats"
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
+)
+
+var _ = Describe(suite.ID(), Label(suite.LabelMetrics, suite.LabelExperimental), Ordered, func() {
```

Comment:
Can you add a TODO comment to move this test to the [metrics_runtime_input_resources_test.go](https://github.com/kyma-project/telemetry-manager/blob/3c3d15ff2eec2da4072ebaa5e6be51723981f73a/test/e2e/metrics_runtime_input_resources_test.go) when the feature flag --k8s-cluster-receiver-allowed is removed